boost-maven-plugin/src/main/java/io/openliberty/boost/liberty/LibertyPackageMojo.java

```diff
@@ -341,9 +327,7 @@ private void installApp(String installAppPackagesVal) throws MojoExecutionExcept
      *
      */
     private void installMissingFeatures() throws MojoExecutionException {
-
-        executeMojo(getPlugin(), goal("install-feature"), configuration(element(name("serverName"), libertyServerName),
-                element(name("features"), element(name("acceptLicense"), "true"))), getExecutionEnvironment());
+        executeMojo(getPlugin(), goal("install-feature"), configuration(element(name("serverName"), libertyServerName)), getExecutionEnvironment());
```

Comment:
Guessing [this](https://github.com/WASdev/ci.maven/issues/381) explains removing the  --acceptLicense.